at_secondary/at_persistence_secondary_server/lib/src/model/at_metadata_builder.dart

```diff
@@ -22,14 +22,15 @@ class AtMetadataBuilder {
       bool? isEncrypted,
       String? dataSignature,
       String? sharedKeyEncrypted,
-      String? publicKeyChecksum}) {
+      String? publicKeyChecksum,
+      String? encoding}) {
     newAtMetaData ??= AtMetaData();
     atMetaData = newAtMetaData;
-    atMetaData.createdAt ??= currentUtcTime;
-    atMetaData.createdBy ??= atSign;
-    atMetaData.updatedBy = atSign;
-    atMetaData.updatedAt = currentUtcTime;
-    atMetaData.status = 'active';
+    atMetaData?.createdAt ??= currentUtcTime;
```

Comment:
Why the question mark `atMetaData?` here?

Reply:
To fix the lint issue, changed the type of atMetadata from var to AtMetadata? (line no. 5). As result, added '?' before atMetadata properties to guard against the null value.

Reply:
You should be using `atMetaData!.createdAt` not `atMetaData?.createdAt` I believe

Reply:
Changed atMetadata?. to atMetadata!.